test/test_files.py

```diff
@@ -585,7 +582,6 @@ def test_unsuccessful_copy(self):
         with pytest.raises(util.FilesystemError):
             util.copy(self.path, self.otherpath)
 
-    @unittest.skipUnless(_common.HAVE_REFLINK, "need reflink")
```

Comment:
This should be `@NEEDS_REFLINK` too, right?

Reply:
I found that this test fails before reflink import is being attempted, see below:

```python
if samefile(path, dest):
    return

if os.path.exists(syspath(dest)) and not replace:
    raise FilesystemError("target exists", "rename", (path, dest))

if fallback:
    with suppress(Exception):
        return import_module("reflink").reflink(path, dest)
...
```

I have now included the `match` parameter in `pytest.raises` to clarify this!

Reply:
Ah, that makes sense.